### PR TITLE
New res flsp demand

### PR DESCRIPTION
**Pull request by jonsampedro**

### CODE REVIEW

File: demand_energy.md [hunk 1]
```diff
@@ -23,10 +23,14 @@ gcam-version: v5.4
 | Name | Resolution | Unit | Source |
 | :--- | :--- | :--- | :--- |
 | Historical energy demand (used for calibration) | By region, technology, and year | EJ/yr | [Exogenous](inputs_demand.html) |
+| Historical floorspace demand (used for calibration) | By region and year | BM2/yr | [Exogenous](inputs_demand.html) |
 | Cost | By region, technology, and year | 1975$/GJ | [Exogenous](inputs_demand.html) |
 | Income and price elasticity | By region, demand, and year | unitless | [Exogenous](inputs_demand.html) |
+| Residential floorspace demand parameters | By region | unitless | [Exogenous](inputs_demand.html) |
+| Satiation levels | By region | m2/pers and EJ/pers | [Exogenous](inputs_demand.html) |
```
Comment (pralitp) [inline demand_energy.md]: I think "resolution" for "Satiation levels" would be better as: "By region and building type / service"

File: inputs_demand.md [hunk 2]
```diff
@@ -99,9 +101,13 @@ Fuel preference elasticities are specified in [A32.fuelprefElasticity.csv](https
 
 Multipliers used to determine the value of time in transit are specified in [A54.tranSubsector_VOTT_revised.csv](https://github.com/JGCRI/gcam-core/blob/master/input/gcamdata/inst/extdata/energy/A54.tranSubsector_VOTT_revised.csv).
 
+#### Residential floorspace parameters
+
+The parameters for the estimation of residential floorspace demand are listed in [A44.flsp_param.csv](https://github.com/JGCRI/gcam-core/blob/master/input/gcamdata/inst/extdata/energy/A44.flsp_param.csv)
```
Comment (pralitp) [inline inputs_demand.md]: I went back and checked the Gompertz CMP: this CSV file doesn't exist.  Instead it looks like the params are fit as part of the data system process (and saved in `L144.flsp_param`).  I'm not sure what to say here then.  Maybe just describe the fitting process and note USA and China have more detailed data sets used?

**Review by pralitp — CHANGES_REQUESTED** [review]
A couple of small fixes are noted.

But also, (unless I missed this in some other documentation PR) don't we need to include the Gompertz functional form in the documentation as well in the `demand_energy.md` / `Equations` section?

File: demand_energy.md (hunk 1 above)
Comment (jonsampedro) [reply demand_energy.md]: OK!

File: inputs_demand.md (hunk 2 above)
Comment (jonsampedro) [reply inputs_demand.md]: Yes, I have explained that the analysis is developed within the model DS, and specified that the values for the US have been estimated outside using subnational data.

**Review by pralitp — APPROVED** [review]

**Pull request merged**